### DISCUSSION TITLE
Connecting to connection pool through Flutter

### DISCUSSION THREAD

**Discussion by tonophant** [supabase/supabase#1862] [Questions] [answered]
How does one use the Flutter SDK to connect to PostgreSQL through the connection pool?  The arguments are the url and the api key but there's no option for the port.  Thanks in advance!

**Comment by awalias** [comment, marked as answer]
hey! if you're connecting via a client library or API it will automatically be pooled, since it connects through the API server [PostgREST](https://postgrest.org) which manages connections itself

you only need to worry about connection pooling if you're connecting to the database directly in which case if you use the port `6543` in your connection string, then it will connect via pgbouncer (`postgres://postgres:password@db.ref.supabase.co:6543/postgres`)

**Reply by tonophant** [reply]
Good to know that it's taken care of automatically. Thanks!
